sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/aio/operations/_operations.py

```diff
@@ -0,0 +1,516 @@
+            request,
+            stream=False,
+            **kwargs
+        )
+        response = pipeline_response.http_response
+
+        if response.status_code not in [200, 202]:
+            map_error(status_code=response.status_code, response=response, error_map=error_map)
+            raise HttpResponseError(response=response)
+
+        deserialized = None
+        response_headers = {}
+        if response.status_code == 200:
+            if response.content:
+                deserialized = response.json()
+            else:
+                deserialized = None
+
+        if response.status_code == 202:
+            response_headers['Operation-Location']=self._deserialize('str', response.headers.get('Operation-Location'))
+            
+
+        if cls:
+            return cls(pipeline_response, deserialized, response_headers)
+
+        return deserialized
+
+
+
+    @distributed_trace_async
+    async def begin_conversation_analysis(
+        self,
+        task: JSON,
+        **kwargs: Any
+    ) -> AsyncLROPoller[JSON]:
+        """Submit analysis job for conversations.
+
+        Submit a collection of conversations for analysis. Specify one or more unique tasks to be
+        executed.
+
+        :param task: The collection of conversations to analyze and one or more tasks to execute.
+        :type task: JSON
+        :keyword str continuation_token: A continuation token to restart a poller from a saved state.
+        :keyword polling: By default, your polling method will be AsyncLROBasePolling. Pass in False
+         for this operation to not poll, or pass in your own initialized polling object for a personal
+         polling strategy.
+        :paramtype polling: bool or ~azure.core.polling.AsyncPollingMethod
+        :keyword int polling_interval: Default waiting time between two polls for LRO operations if no
+         Retry-After header is present.
+        :return: An instance of AsyncLROPoller that returns JSON object
+        :rtype: ~azure.core.polling.AsyncLROPoller[JSON]
+        :raises: ~azure.core.exceptions.HttpResponseError
+
+        Example:
+            .. code-block:: python
+
+                # JSON input template you can fill out and use as your body input.
+                task = {
+                    "analysisInput": {
+                        "conversations": [
+                            {
+                                "domain": "str",  # Optional. Enumeration of
+                                  supported conversational domains. Known values are: "finance",
+                                  "healthcare", "generic".
+                                "id": "str",  # Required. Unique identifier for the
+                                  conversation.
+                                "language": "str",  # Required. The language of the
+                                  conversation item in BCP-47 format.
+                                modality: modality
```

Comment:
Something happened here as well... I think `modality` breaks this from being copy/paste-able. @iscai-msft